src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.getElementById('largeImg');
+
+const list = document.querySelector('#thumbs');
+
+list.addEventListener('click', (eve) => {
+  const link = event.target.closest('img');
```

Comment:
There is a reference error here. You should use `eve.target` instead of `event.target` since `eve` is the event object passed to the event listener.